### PR TITLE
Fix type erasure

### PR DESCRIPTION
**Pull request by imjasonmiller**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,64 +1,7 @@
-import GlobeScene from './components/scene';
-import GeoJSONType from './enums/geojson';
-import Marker from './components/marker';
-import Bar from './components/bar';
+import GlobeScene from "./components/globe";
+import Arc from "./components/arc";
+import Marker from "./components/marker";
+import Bar from "./components/bar";
+import Globe from "./components/globe";
```
Comment (Wouter125) [inline src/index.ts:5]:
```suggestion
```

Internal use only.

File: src/index.ts [hunk 2]
```diff
@@ -1,64 +1,7 @@
-import GlobeScene from './components/scene';
-import GeoJSONType from './enums/geojson';
-import Marker from './components/marker';
-import Bar from './components/bar';
+import GlobeScene from "./components/globe";
```
Comment (Wouter125) [inline src/index.ts:1]:
```suggestion
import GlobeScene from "./components/scene";
```

The scene needs to be imported from scene, not globe.

File: src/types/geojson.ts [hunk 3]
```diff
@@ -1,4 +1,11 @@
-import GeoJSONType from '../enums/geojson';
+export enum GeoJSONType {
```
Comment (Wouter125) [inline src/types/geojson.ts:1]: An enum is not a type right. We do this because we moved it from a types file to a ts file right?

**Review by Wouter125 — CHANGES_REQUESTED** [review]

File: src/types/geojson.ts (hunk 3 above)
Comment (imjasonmiller) [reply src/types/geojson.ts:1]: It is a type, an enumerated type! 😄 The name somewhat threw me off, but it's the property for the `type` key, hence `Type`

My reasoning was that for the code base in its current size it's easier to keep it close to its related (GeoJSON) components, as I had to look for a bit while resolving related errors.

**Pull request merged**